Tests/SwiftSyntaxBuilderTest/StructTests.swift

```diff
@@ -45,7 +45,8 @@ final class StructTests: XCTestCase {
         .carriageReturns(1),
       ],
       structKeyword: .keyword(.struct),
-      identifier: "CarriateReturnsStruct"
+      identifier: "CarriateReturnsStruct",
+      membersBuilder: {}
```

Comment:
It’s not really important but for the empty cases, I would not use the result builders, it somehow seems cleaner to me that way.
```suggestion
      members: MemberDeclBlockSyntax(members: [])
```

Same for the other test cases.